themes/10up-theme/assets/css/frontend/tailwind/index.css

```diff
@@ -0,0 +1,47 @@
+/* stylelint-disable at-rule-no-unknown */
```

Comment:
This is a far better approach - nice work - do we still need the above stylelint comment if we've amended the stylelint config?

Reply:
I imagine we wouldn't need both, but I think the discrete file is more scalable for sure.